requirements.txt

```diff
@@ -67,4 +67,4 @@ whitenoise==4.1
 wrapt==1.10.11
 Django==2.1.2
 django-autoslug-iplweb==1.9.4
-django-versatileimagefield==1.10
+django-versatileimagefield==1.10
```

Comment:
This is change is unnecessary considering there's no difference from the first dependency.

Reply:
I have removed it. Kindly review it again.